plugins/json/core/src/test/java/org/pentaho/di/trans/steps/jsoninput/analyzer/JsonInputAnalyzerTest.java

```diff
@@ -196,4 +196,17 @@ public void testJsonInputExternalResourceConsumer() throws Exception {
 
     assertEquals( JsonInputMeta.class, consumer.getMetaClass() );
   }
+  
+  @Test
+  public void testCloneAnalyzer() {
+    final JsonInputAnalyzer analyzer = new JsonInputAnalyzer();
+    // verify that cloneAnalyzer returns an instance that is different from the original
+    assertNotEquals( analyzer, analyzer.cloneAnalyzer() );
+  }
+
+  @Test
+  public void testNewInstance(){
```

Comment:
This test isn't really needed - if it was added for coverage, I think the new added testCloneAnalyzer might do the trick. It doesn't hurt though, so we can leave it here.... Also, just a nit-picky comment, there's a missing space before the last closing parenthesis ;) - I'm guessing checkstyle isn't run on this test code.